docs/optimizations/FileLocalization.md

```diff
@@ -28,6 +28,10 @@ None!
 
 Files marked for optional localization are still treated in exactly the same way as other `File` inputs for call caching.
 
+## Cromwell Support
+
+This optimization will be active in Cromwell as of version 33 and higher.
```

Comment:
I recommend putting this and all notes about Cromwell version support at the top of the file. This way a user sees the availability before reading through the rest of the doc.

Something like:
> Available in Cromwell version 33 and higher.